front-end/.gitignore

```diff
@@ -9,6 +9,9 @@
 # production
 /build
 
+# css
+src/**/*.css
```

Comment:
why are we ignoring css?

Reply:
We use sass in our project, it builds the css when compiling, and the referencing in the react components is done as css. So, we don't really need to track them.

Reply:
I think it is better to have webpack do this for us than direct compiling